README.md

```diff
@@ -67,6 +67,7 @@ git clone \
 #### Install and Build
 
 ```bash
+DETECT_CHROMEDRIVER_VERSION=true yarn global add chromedriver
```

Comment:
This should go after `nvm use` or else you might install it to a different nodejs. Also a comment would be nice that version of global chromedriver installation has to be maintained to match the installed Google Chrome version